backend/src/main/java/com/carffeine/carffeine/domain/chargestation/congestion/PeriodicCongestion.java

```diff
@@ -31,8 +34,8 @@ public class PeriodicCongestion {
 
     @ManyToOne
     @JoinColumns({
-            @JoinColumn(name = "station_id", referencedColumnName = "station_id", insertable = false, updatable = false),
-            @JoinColumn(name = "charger_id", referencedColumnName = "charger_id", insertable = false, updatable = false)
+            @JoinColumn(name = "station_id", referencedColumnName = "station_id", insertable = false, updatable = false, foreignKey = @ForeignKey(value = ConstraintMode.NO_CONSTRAINT)),
+            @JoinColumn(name = "charger_id", referencedColumnName = "charger_id", insertable = false, updatable = false, foreignKey = @ForeignKey(value = ConstraintMode.NO_CONSTRAINT))
```

Comment:
```suggestion
  @JoinColumns(value = {
            @JoinColumn(name = "fk_station_id"),
            @JoinColumn(name = "fk_charger_id")
    }, foreignKey = @ForeignKey(ConstraintMode.NO_CONSTRAINT))
```

이렇게 하면 한번에 ForeignKey 를 해제할 수 있어요